### DISCUSSION TITLE
Ready to be publicised?

### DISCUSSION THREAD

**Discussion by spookylukey** [AliSayyah/django-urlconfchecks#32] [General]
Hi @AliSayyah 

I referenced the prototype code for this package on this page - https://spookylukey.github.io/django-views-the-right-way/url-parameters.html#discussion-type-checked-parameters

If you think this is ready for a wider audience, I'll replace that with a link to this package, and tweet about it. I'm very impressed with the quality of the package you've created!

If you get more usage, you'll also get bug reports, which is all fine while you're having fun, but can get quite tiring quite quickly, so I don't want to send anything your way until you are ready for it.

**Comment by AliSayyah** [comment]
Wow, thank you!
That would be nice.
Yes, I think it's ready.
I wanted to extend the tests slightly but, If you think it's in good shape, I'm all in for publicizing.

**Reply by spookylukey** [reply]
Great - I tweeted here - https://twitter.com/spookylukey/status/1491497068040491009 and added a link here - https://spookylukey.github.io/django-views-the-right-way/url-parameters.html#discussion-type-checked-parameters

**Reply by AliSayyah** [reply]
@spookylukey, thank you very much.
